Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL := /bin/bash
+
+CURRENT_PATH = $(shell pwd)
+REPO = # ghcr.io
+IMAGE = techneatium/techs-checks-latex
+TAG = latest
+FULL_IMAGE = ${REPO}${IMAGE}:${TAG}
+TARGET = F14_CheatSheet.tex
+
+.PHONY: build_container
+prepare_image: clean_fonts
+	docker build -t ${FULL_IMAGE} .
+
+.PHONY: build_pdf
+build_pdf:
+	mkdir -p ./build
+	source scripts/set_build_info.sh
+	docker run --rm -it \
+		-v ${CURRENT_PATH}:/source:rw,Z \
+		--entrypoint /bin/bash ${FULL_IMAGE} -c "\
+		cd /source &&\
+		scripts/set_build_info.sh &&\
+		cp -v /fonts/* /source/. &&\
+		lualatex --output-format=pdf --output-folder=/source/build ${TARGET} ;\
+		lualatex --output-format=pdf --output-folder=/source/build ${TARGET} ;\
+		lualatex --output-format=pdf --output-folder=/source/build ${TARGET}"
```

Comment:
`--output-folder= ...` Does not seem to have any effect for lualatex. As far as I am aware the command should be ` --output-directory= ...`, 

However, when this is run several errors are encountered as the main .tex file cannot locate files such as the titlepage. This might require reorganizing the folder structure

Reply:
I've removed the `--output-folder`, and coupled with the forced cleanups before and after the runs it should be a non-concern.